resources/definitions/teraform_inventor.def.json

```diff
@@ -0,0 +1,149 @@
+  {
+    "version": 2,
+    "name": "TeraForm Inventor",
+    "inherits": "fdmprinter",
+    "metadata":
+    {
+        "visible": true,
+        "author": "SpaceFactory",
```

Comment:
I can't find anything about this printer. Is this a printer that is commercially available? If not, there isn't really a reason to add it to the distribution.

Reply:
I think it might be? Their website ( https://www.aispacefactory.com/about ) mentions how they 'develop advanced construction technologies', so that seems to imply that they won't just keep the hardware in-house.

Reply:
Yes - printer will be commercially available next month.

Reply:
We don't normally publish profiles for printers that have not yet been published, not even if the contributor promises out of the good of their hearts that they will soon publish the printer.